src/Makevars.in

```diff
@@ -86,7 +95,18 @@ endif
 all: tbb $(SHLIB)
 
 tbb:
+ifdef TBB_LIB
+	echo "Using system (Intel/OneAPI) TBB library..."; \
 	mkdir -p ../inst/lib/$(ARCH_DIR); \
+	cp $(TBB_LIB)/libtbb.so ../inst/lib/$(ARCH_DIR)/libtbb.so; \
+	cp $(TBB_LIB)/libtbbmalloc.so ../inst/lib/$(ARCH_DIR)/libtbbmalloc.so; \
+	rm -rf ../inst/include/serial/ ../inst/include/tbb/; \
+	rm -rf ../inst/include/tbb_local/
+else
+	echo "Building TBB library from source code..."; \
+	mkdir -p ../inst/lib/$(ARCH_DIR); \
+	cp -r ../inst/include/tbb_local/* ../inst/include/; \
```

Comment:
```suggestion
	cp -R ../inst/include/tbb_local/* ../inst/include/; \
```

Huge nitpick, but technically `-r` is declared obsolete by POSIX and can have implementation-defined behavior.